src/routes/home/HomeScreen.tsx

```diff
@@ -48,16 +49,25 @@ const HomeScreen = ({ navigation }: HomeScreenProps): JSX.Element => {
     navigation.navigate('AddCustomDiscipline')
   }
 
+  const navigateToDisciplineSelection = (discipline: Discipline): void => {
+    navigation.navigate('DisciplineSelection', { discipline })
+  }
+
   const customDisciplineItems = customDisciplines?.map(customDiscipline => (
     <DisciplineCard
       key={customDiscipline}
       identifier={{ apiKey: customDiscipline }}
       refresh={refreshCustomDisciplines}
+      onPress={navigateToDisciplineSelection}
     />
   ))
 
   const selectedProfessionItems = selectedProfessions?.map(profession => (
-    <DisciplineCard key={profession} identifier={{ disciplineId: profession }} />
+    <DisciplineCard
+      key={profession}
+      identifier={{ disciplineId: profession }}
+      onPress={navigateToDisciplineSelection}
```

Comment:
Not sure if there is a reason why this was removed in the first place, without this it is not possible to view the vocabulary list or select a specific exercise/discipline.